application/CohortManager/src/Functions/CohortDistributionServices/TransformDataService/TransformDataServiceConfig.cs

```diff
@@ -12,4 +12,8 @@ public class TransformDataServiceConfig
     public string BsSelectGpPracticeUrl { get; set; }
     [Required]
     public string LanguageCodeUrl { get; set; }
+    [Required]
+    public string ExcludedSMULookupUrl { get; set; }
+
+    public string CacheTimeOutHours { get; set; }
```

Comment:
please add a default and make an int?
```suggestion
    public int CacheTimeOutHours { get; set; } = 24;
```